Gopkg.lock

```diff
@@ -26,16 +32,24 @@
   ]
   revision = "e09c5db296004fbe3f74490e84dcd62c3c5ddb1b"
 
+[[projects]]
+  name = "github.com/google/uuid"
+  packages = ["."]
+  revision = "064e2069ce9c359c118179501254f67d7d37ba24"
```

Comment:
What is the reason to define `revision` in conjunction with `version`?

Reply:
I needed to update the toolkit to version 0.3.0. The changes to the `Gopkg.lock` result directly from `dep` resolving the following command:
```
dep ensure -update github.com/infobloxopen/atlas-app-toolkit
```